Login.php

```diff
@@ -0,0 +1,49 @@
+<?php
+declare(strict_types=1);
+session_start();
+
+require_once 'src/models/UsersModel.php';
+require_once 'src/services/DatabaseConnectionService.php';
+
+$db = DatabaseConnectionService::connect();
+
+if (!isset($_SESSION['loggedIn'])) {
```

Comment:
Suggestion!

You can simplify this code somewhat if you reverse this check.

I would check to see if they are logged in, if so, redurect to index.php

Then you can have your if statement to check the form has been submitted after. That reduces the mount of nesting you have to do.

EG:

```
if (they are logged in) {
    header('.....')
    exit();
}

if (isset($_POST[...]) {

}
```